leaderboard/forms.py

```diff
@@ -21,10 +23,38 @@ class PublicationNameForm(forms.Form):
     Based on forms.Form as we don't want to create a new Team.
     """
 
+    institution_name = forms.CharField(
+        max_length=MAX_NAME_LENGTH,
+        validators=[validate_institution_name],
+        widget=forms.TextInput(attrs={'class': 'form-control'}),
+        label='Institution name',
+    )
     publication_name = forms.CharField(
         max_length=MAX_NAME_LENGTH,
-        validators=[validate_team_name],
+        validators=[validate_publication_name],
         widget=forms.TextInput(attrs={'class': 'form-control'}),
+        label='Short team name',
+    )
+
+
+class PublicationDescriptionForm(forms.Form):
+    """Form used for teampage view.
+
+    Based on forms.Form as we don't want to create a new Team.
+    """
+
+    publication_url = forms.CharField(
+        max_length=MAX_NAME_LENGTH,
+        widget=forms.TextInput(attrs={'class': 'form-control'}),
+        label='System paper',
+        required=False,
+    )
+    description = forms.CharField(
+        max_length=MAX_DESCRIPTION_LENGTH,
+        widget=forms.Textarea(
+            attrs={'class': 'form-control', 'rows': 6, 'cols': 80}
```

Comment:
@snukky: `80` seems like a retro value ;)

Reply:
You're right, removed.